test_util/test_upgrade_vpc.py

```diff
@@ -271,7 +271,8 @@ def test_app_dns_survive_upgrade():
             self.log_test("test_upgrade_vpc.test_app_dns_survive_upgrade", test_app_dns_survive_upgrade)
 
     def run_test(self) -> int:
-        stack_name = 'upgrade-test-' + random_id(10)
+        stack_prefix = os.getenv("CF_STACK_NAME_PREFIX", "dcos-upgrade-test")
```

Comment:
This will allow you to deploy things that will not get cleaned by cloud cleaner. We are using generated keys now and want to use the right magic strings everywhere (generated keys are named with the stack name). https://github.com/dcos/dcos/pull/1238/files#diff-be106601b4bf185940d95c6bce4a625bR270

So, instead, just make this so that dcos-ci-test is still used, e.g.
```stack_prefix = os.getenv('CF_STACK_NAME_PREFIX', '')
stack_name = f"{stack_prefix}-dcos-ci-test"```